n3fit/src/n3fit/tests/test_evolven3fit.py

```diff
@@ -145,11 +146,14 @@ def test_eko_utils(tmp_path):
     assert_allclose(list(eko_op.operator_card.raw["mugrid"]), op_card_dict["mugrid"])
 
 
-@pytest.mark.parametrize("fitname", ["Basic_runcard_3replicas_lowprec_399"])
+@pytest.mark.parametrize("fitname", ["Basic_runcard_3replicas_lowprec_399", "Basic_runcard_qed_3replicas_lowprec_398"])
 def test_perform_evolution(tmp_path, fitname):
     """Test that evolven3fit_new is able to utilize the current eko in the respective theory.
     In addition checks that the generated .info files are correct
     """
+    theory = API.theoryid(theoryid=int(fitname[-3:]))
+    # check that nothing went wrong
+    assert isinstance(theory, TheoryIDSpec)
```

Comment:
The output of the function is always a `TheoyrIDSpec`.  Also, this doesn't really test something specific for evolevn3fit - in fact, tests of the `check_theoryid` function exist elsewhere